src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -490,7 +491,15 @@ fun RsMod.importItem(candidate: ImportCandidate) {
         0 -> "self::"
         else -> "super::".repeat(relativeDepth)
     }
-    insertUseItem(psiFactory, "$prefix${info.usePath}")
+
+    val insertionScope = if (context.isDoctestInjection) {
```

Comment:
Could you add a comment about different cases and add short examples (with doc test with and without module)?

Reply:
Or you can create testmark for each case and "link" the corresponding tests with this logic
Looks like it allows you not to write any examples because you've already added them in tests